lib/language_pack/installers/heroku_ruby_installer.rb

```diff
@@ -22,5 +22,20 @@ def fetch_unpack(ruby_version, install_dir, build = false)
       @fetcher.fetch_untar(file)
     end
   end
+
+  def setup_binstubs(*)
+    super
+    Dir["#{DEFAULT_BIN_DIR}/{rake,bundle,rails}"].select do |binstub|
+      begin
+        if File.file?(binstub)
+          shebang = File.open(binstub, &:readline)
+          if !shebang.match %r{\A#!/usr/bin/env ruby(.exe)?\z}
+            warn("Binstub #{binstub} contains shebang #{shebang}. This may cause issues if the program specified is unavailable.", inline: true)
```

Comment:
I would like a link to that devcenter article you wrote here. If you give me the link I can edit it and give some feedback as well.